src/routes/get-started/ReportBug.tsx

```diff
@@ -10,13 +10,34 @@ import {
   GoATextArea
 } from "@abgov/react-components";
 import { ComponentContent } from "@components/component-content/ComponentContent.tsx";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { Octokit } from "octokit";
 import { useNavigate } from "react-router-dom";
   
 export default function ReportBugPage() {
+  interface IPackage {
```

Comment:
Not a big issue, but no need for the  C# style of defining interfaces. `Package` is fine

Reply:
Fair, just how I'm used to defining interfaces, done.